src/flint/fq_poly.jl

```diff
@@ -659,6 +657,11 @@ function sub!(z::FqPolyRepPolyRingElem, x::FqPolyRepPolyRingElem, y::FqPolyRepPo
   return z
 end
 
+function reverse!(z::FqPolyRepPolyRingElem, x::FqPolyRepPolyRingElem, len::Int)
```

Comment:
Since there is a heading "Reversal" that contains the `reverse` function, I would find it easier for future reference if `reverse!` would be located there as well, instead of under "Mutable arithmetics" (same for all other `*_poly` files)

Reply:
Fine by me. All I want is a somewhat semi-predictable place where to find stuff